packages/next/src/shared/lib/image-blur-svg.ts

```diff
@@ -16,20 +16,21 @@ export function getImageBlurSvg({
   blurDataURL: string
   objectFit?: string
 }): string {
-  const std = blurWidth && blurHeight ? '1' : '20'
+  const std = blurWidth && blurHeight ? 0.5 : 20
   const svgWidth = blurWidth || widthInt
   const svgHeight = blurHeight || heightInt
-  const feComponentTransfer = blurDataURL.startsWith('data:image/jpeg')
-    ? `%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='1 1'/%3E%3C/feComponentTransfer%3E%`
+
+  const svgDimensions = svgWidth && svgHeight
+  const svgSizeAttributes = svgDimensions
```

Comment:
```suggestion
  const viewBox = svgWidth && svgHeight
```

I think you can remove the `svgDimensions` variable since the viewbox is falsy